ooiui/templates/science/plotting.html

```diff
@@ -535,11 +535,10 @@
       parameterhtml += "</optgroup>"
 
       $("div#yvar0-selection-default > div.form-group > select").html(parameterhtml);
-      if (refreshYVar1){
-        $("div#yvar1-selection > div.form-group > select").html(parameterhtml);
-      }
```

Comment:
@Bobfrat we just always refresh the list now as before it was not getting refreshed for the other plot types